packages/office-ui-fabric-react/src/components/Button/Button.types.ts

```diff
@@ -504,9 +504,4 @@ export interface IButtonStyles {
    * Style override for the SplitButton FlexContainer.
    */
   splitButtonFlexContainer?: IStyle;
-
-  /**
-   * Style override for focus on the the SplitButton Container
-   */
-  splitButtonFlexContainerFocus?: IStyle;
```

Comment:
Do we need to deprecate this as it is a change in api?

Reply:
@chang47 can correct me if I'm wrong, but I think this was added a week ago by Josh and I would imagine it's pretty safe to assume we were the only ones that have used it.

Reply:
If that's the case I think we're good.

Reply:
@joschect: @jspurlin is correct, this was something that I added about a week ago, but decided that it was bad design and made this PR to rectify it.